### ISSUE TITLE
New Rule: `no-store`

### ISSUE THREAD

**Issue by baseballyama** [sveltejs/eslint-plugin-svelte#813]
### Motivation

Svelte 5 allows state management using runes. On the other hand, we can also manage state using store. This rule prohibits the use of store, thereby restricting state management to runes only.

### Description

DO not use store. Please use runes instead.

### Examples

```js
<!-- ✓ GOOD -->
let count = $state(0);

/* ✗ BAD */
import { writable, readable, derived } from 'svelte/store';
```

**Comment by baseballyama** [comment]
duplicate of https://github.com/sveltejs/eslint-plugin-svelte/issues/615
